tests/codeception/e2e/app/LoginCest.php

```diff
@@ -0,0 +1,37 @@
+<?php
+/**
+ * @group mandatory
+ */
+class LoginCest
+{
+    public function _before(E2eTester $I)
+    {
+    }
+
+    public function _after(E2eTester $I)
+    {
+    }
+
+    // tests
+    public function tryToTest(E2eTester $I)
```

Comment:
all tests are named `tryToTest`, could you give them a more meaningful name, here eg. `frontendLogin`, usually the string in `wantTo` is  good guide.